client/src/components/nav-bar/NavBar.tsx

```diff
@@ -0,0 +1,222 @@
+  ];
+
+  /**
+   * Handles the navigation click event. Updates the selected index and navigates to the new path.
+   *
+   * @param {number} newValue - The index of the selected item.
+   * @param {string} path - The route path to navigate to.
+   */
+  const handleNavigation = (newValue: number, path: string) => {
+    setValue(newValue);
+    navigate(path);
+  };
+
+  // desktop view (Vertical Sidebar)
+  if (isSidebarMode) {
+    return (
+      <Box
+        sx={{
+          width: "100%",
+          zIndex: 1000,
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "center",
+          py: 2,
+          ml: 2,
+        }}
+      >
+        {/* Iterate over navigation items */}
+        {navItems.map((item, index) => (
+          <Box
+            key={item.label}
+            onClick={() => handleNavigation(index, item.path)}
+            sx={{
+              width: 60,
+              height: 75,
+              display: "flex",
+              flexDirection: "column",
+              alignItems: "center",
+              justifyContent: "center",
+              cursor: "pointer",
+              mb: 1,
+              bgcolor: "transparent",
+              color: theme.palette.text.secondary,
+              WebkitTapHighlightColor: "transparent",
+              userSelect: "none",
+            }}
+          >
+            {/* Icon Wrapper (Handles the active background style) */}
+            <Box
+              sx={{
+                display: "flex",
+                alignItems: "center",
+                justifyContent: "center",
+
+                ...(value === index && {
+                  width: 50,
+                  height: 40,
+                  borderRadius: 999,
+                  backgroundColor: theme.palette.secondary.light,
+                }),
+                mb: 0.5,
+              }}
+            >
+              {/* Render the icon component */}
+              <item.icon
+                sx={{
+                  fontSize: 24,
+                  color:
+                    value === index
+                      ? "#FFFFFF !important"
+                      : theme.palette.text.secondary,
+                }}
+              />
+            </Box>
+            {/* Navigation Label */}
+            <Typography
+              variant="caption"
+              sx={{
+                fontWeight: "bold",
+                color:
+                  value === index
+                    ? theme.palette.text.secondary
+                    : theme.palette.text.secondary,
+              }}
+            >
+              {item.label}
+            </Typography>
+          </Box>
+        ))}
+      </Box>
+    );
+  }
+
+  // mobile view (Horizontal Bottom Bar)
+  return (
+    <Paper
+      sx={{
+        position: "fixed",
+        bottom: 20,
+        left: "50%",
+        width: "80%",
+        transform: "translateX(-50%)",
+        borderRadius: 50,
+        zIndex: 1000,
+      }}
+      elevation={3}
```

Comment:
```suggestion
      elevation={0}
```
Den Schatten vielleicht entfernen, dann sieht es aus wie im Prototyp und die Card ist mehr betont